django/utils/safestring.py

```diff
@@ -22,6 +22,8 @@ class SafeBytes(bytes, SafeData):
     """
     A bytes subclass that has been specifically marked as "safe" (requires no
     further escaping) for HTML output purposes.
+    Kept in Django 2.0 for backwards-compatibility, but should not be actively
```

Comment:
suggested wording: "Kept in Django 2.0 for usage by apps supporting Python 2. Shouldn't be used in Django anymore."